src/styles/index.scss

```diff
@@ -2,2 +1,152 @@
+.body {
   margin: 0;
 }
+
+.button {
+  width: 166px;
+  height: 40px;
+  border-radius: 5px;
+  margin: 16px;
+  background: #00acdc;
+  border: 0;
+  align-items: center;
```

Comment:
Property `align-items` should be used within a flex or grid container. Since `.button` is not specified as a flex or grid container, this property has no effect here.